src/services/ReportHub/ReportHub.BLL/Extensions/MassTransitExtensions.cs

```diff
@@ -0,0 +1,53 @@
+using System.Reflection;
+using MassTransit;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Shared.SharedModels;
+using ReportHub.BLL.MassTransit.Consumers;
+
+namespace ReportHub.BLL.Extensions
+{
+    public static class MassTransitExtensions
+    {
+        public static void ConfigureMassTransit(
+            this IServiceCollection services,
+            IConfiguration config
+        )
+        {
+            services.AddMassTransit(x =>
```

Comment:
Correct the naming in all services with MassTransit configuration. Make the naming more understandable